egov-indexer/egov-bpa-indexer.yml

```
@@ -2,277 +2,277 @@ ServiceMaps:
+                    outJsonPath: $.Data.history
+    - topic: bpa-connection-legacyIndex
+      configKey: LEGACYINDEX
+      indexes:
+        - name: bpa-index
+          type: bpaapplication
+          id: $.id, $.tenantId
+          isBulk: true
+          jsonPath: $.BPA.*
+          timeStampField: $.auditDetails.createdTime
+          customJsonMapping:
+            indexMapping: {"Data":{ "workflow":{"action":"",  "assignes":[ ]},"businessService":"businessService","edcrNumber":"edcrNumber","riskType":"riskType","applicationNo":"applicationNo","approvalNo":"approvalNo","approvalDate":"approvalDate","landInfo":{},"applicationDate":"applicationDate","status":"status","tenantId":"tenantId","@timestamp":"timestamp", "serviceType":"", "applicationType":"","auditDetails":{},"plotAreaApproved":"plotAreaApproved"}}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)